tests/drivers/ipm/testcase.yaml

```diff
@@ -2,5 +2,5 @@ tests:
   peripheral.mailbox:
     filter: not CONFIG_SOC_QUARK_SE_C1000_SS
     arch_exclude: posix xtensa
-    platform_exclude: qemu_x86_64 # see issue #12478
+    #platform_exclude: qemu_x86_64 # see issue #12478
```

Comment:
Please remove this line if you're fixing the reason for the exclude, don't comment it out.

Reply:
yes, I will remove it.